src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/IPublicationRepository.cs

```diff
@@ -8,7 +8,9 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces
 {
     public interface IPublicationRepository
     {
-        Task<List<Publication>> GetAllPublicationsForTopic(Guid topicId);
+        Task<List<Publication>> GetPublicationsForTopic(Guid topicId);
+
+        Task<List<Publication>> GetPublicationListForUser(Guid userId, Guid? topicId = null);
```

Comment:
> Is this ever going to be used without a topic? I would just call this GetPublicationsByTopicAndUser and get rid of the optional param.

The intention is for the method to return all publications associated with a user if no topic argument is provided. I was following Nick's spec with this.

> What's the difference between this and the method below it called GetPublicationsForTopicRelatedToUser? It's too confusing to have both.

I copied `GetPublicationForTopicRelatedToUser` to create `ListPublicationsForTopicId`. The idea was that `GetPublicationsForTopicRelatedToUser` is going to get deleted, but for now I'll just create a copy that is totally separate. But I think you're right that since both methods basically behave the same, I can remove `GetPublicationsForTopicRelatedToUser`.